vaxrank/cli.py

```diff
@@ -218,13 +225,15 @@ def check_args(args):
             args.output_html_report or
             args.output_pdf_report or
             args.output_json_file or
-            args.output_xlsx_report):
+            args.output_xlsx_report or
```

Comment:
Ha, eventually we'll have to do this a little more elegantly but I don't currently have any better ideas.

Reply:
Agreed: this kind of Frankensteined itself, was initially meant to be a small pretty-printed thing but now we're using it for all kinds of outputs. Eventually worth refactoring